src/components/EnvelopeList.vue

```diff
@@ -92,14 +92,15 @@
 				class="icon-loading-small"
 				:class="{refreshing: refreshing}" />
 			<Envelope
-				v-for="env in envelopes"
+				v-for="(env, index) in envelopes"
 				:key="env.databaseId"
 				:data="env"
 				:mailbox="mailbox"
-				:selected="isEnvelopeSelected(envelopes.indexOf(env))"
+				:selected="isEnvelopeSelected(index)"
```

Comment:
WHat happens if a new message arrives? Index will shift?
Is index the correct keyword here or is this a uuid?

Reply:
As far as I remember:

I don't think we manage it yet when new messages arrive.
And, yes it's an index, not a message id.... Maybe not the wiser choice though

Reply:
it's the old issue from https://github.com/nextcloud/mail/pull/2619

time to fix this in a follow-up PR :wink: